src/applications/personalization/dashboard/reducers/folders.js

```diff
@@ -66,6 +70,17 @@ export default function folders(state = initialState, action) {
       );
     }
 
+    case FETCH_FOLDER_FAILURE: {
+      return set(
+        'data.currentItem',
+        {
+          errors: action?.errors,
+          loading: false,
```

Comment:
should this be `fetching: false`?

Reply:
Also weird that down on line 100, `fetching: true` is being set on the `state.ui` instead of `state.currentItem` 🤔  Something weird is going on here

Reply:
Oh line 100 isn't needed at all. Line 87 will effectively set `fetching: true`